Makefile

```diff
@@ -29,10 +29,10 @@ check: ## Checks code for linting/construct errors
 	@echo "    [✓]\n"
 build: ## Builds the docker-compose environment
 	@echo "==> Building docker image..."
-	@docker-compose $(docker_files) build
+	@docker-compose $(docker_files) build --no-cache
```

Comment:
`--no-cache` and `--force-recreate` have different purposes and effects. `--no-cache` will make the build longer but not reusing cached layers of the image. `--force-recreate` will make sure that the containers are recreated. First one affects the built image(s). Second one affects the launched containers.